src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -36,8 +41,21 @@ const defaultProps = {
 };
 
 const ReportActionItemCreated = (props) => {
+    const participants = lodashGet(props.report, 'participants', []);
+    const isChatRoom = ReportUtils.isChatRoom(props.report);
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
+
+    function navigateToDetailsPage() {
+        if (isChatRoom || isPolicyExpenseChat) {
+            return Navigation.navigate(ROUTES.getReportDetailsRoute(props.report.reportID));
+        }
+        if (participants.length === 1) {
+            return Navigation.navigate(ROUTES.getDetailsRoute(participants[0]));
+        }
+        Navigation.navigate(ROUTES.getReportParticipantsRoute(props.report.reportID));
+    }
+
```

Comment:
This method is pretty much the same as the code here

https://github.com/Expensify/App/blob/cc14895f68cf2acfd048f06396fbed0de808853e/src/pages/home/HeaderView.js#L128-L136

Can we make it a util function?
cc: @chiragsalian

Reply:
Any update @Tushu17?

Reply:
Missed the earlier notification.
Few questions:-

1. What should be the name of the util function? Is `navigateToDetailsPage` fine?
2. Should add it inside reportUtils.js ?
3. What should be at `@returns {}`? Should it be `@returns {void}`

Reply:
> 1. What should be the name of the util function? Is `navigateToDetailsPage` fine?

Yes looks fine to me.
> 2. Should add it inside reportUtils.js ?

Yes

> 3. What should be at `@returns {}`? Should it be `@returns {void}`

return is not needed here.

cc: @chiragsalian

Reply:
@Santhosh-Sellavel, if I don't add `@retruns` I get lint error "Missing JSDoc @returns for function.eslint"
but I can use // eslint-disable-next-line valid-jsdoc at the top of this function to disable it or I can use @returns {void}

Reply:
Write like this ..
```js
Navigation.navigate(ROUTES.getDetailsRoute(participants[0]));
return;
```

Reply:
Made the changes!